AstarteDeviceSDKCSharp/Transport/MQTT/AstarteMqttV1Transport.cs

```diff
@@ -154,5 +154,42 @@ private int QosFromReliability(AstarteInterfaceDatastreamMapping mapping)
             }
         }
 
+        public override async Task ResendAllProperties()
+        {
+            if (_astartePropertyStorage.Equals(null))
```

Comment:
Can `_astartePropertyStorage` be null? in AstarteMqttV1Transport base class was declare as `non-nullable` reference... Or am I getting it wrong?